tensorflow_lite_support/c/task/vision/image_segmenter.h

```diff
@@ -99,13 +99,14 @@ typedef struct TfLiteImageSegmenter TfLiteImageSegmenter;
 // perform on the raw model results (see TfLiteSegmentationResult for more).
 enum TfLiteImageSegmenterOutputType {
   kUnspecified,
-  // Gives a single output mask where each pixel represents the class which
-  // the pixel in the original image was predicted to belong to.
-  kCategoryMask,
 
   // Gives a list of output masks where, for each mask, each pixel represents
   // the prediction confidence, usually in the [0, 1] range.
-  kConfidenceMask
+  kCategoryMask,
```

Comment:
You should reverse Confidence Mask and Category Mask here. This is a fix that I added in the previous merged pull request.

Reply:
I have kept the order consistent with C++ port since in that way C enum can be cast to C++ enum in a single line rather than doing an if condition check for each value in the enum. LMK if that's preferred.

Reply:
Is the order you modified to kUnspecified, kCategoryMask, kConfidenceMask? That's the order even I had in mind.

Reply:
I just meant that the comments for kCategoryMask should be for kConfidenceMask and vice versa. It looks like you've fixed it now.